subsys/power/device.c

```diff
@@ -29,6 +29,15 @@ static const char *const core_devices[] = {
 	"sys_clock",
 	"UART_0",
 };
+#endif
+
+#if defined(CONFIG_SOC_SERIES_KINETIS_K6X)
+#define MAX_PM_DEVICES		1
+#define MAX_DEV_NAME_LEN	16
+static const char core_devices[][MAX_DEV_NAME_LEN] = {
```

Comment:
I've seen this pattern before; is there any reason why `MAX_DEV_NAME_LEN` must be reserved for each entry?  This could be:
```
static const char *core_devices[] = {
    DT_ETH_MCUX_0_NAME,
}
```
which would save a few bytes and allow string constant sharing.

Reply:
That's how this PM awfulness was made. It's not the point of this PR to change anything there.

Reply:
wait, that was changed recently, that's a rebasing left-over.